### PR TITLE
Update external setup guide

### PR DESCRIPTION
**Pull request by oxve**
b/286653238

### CODE REVIEW

File: cobalt/site/docs/development/setup-linux.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ Required libraries can differ depending on your Linux distribution and version.
 
     ```
     $ sudo apt update && sudo apt install -qqy --no-install-recommends \
-        pkgconf ninja-build bison nasm yasm binutils clang libgles2-mesa-dev \
-        mesa-common-dev libpulse-dev libavresample-dev libasound2-dev \
-        libxrender-dev libxcomposite-dev libxml2-dev curl git \
-        python3.8-venv libxi-dev
+        pkgconf ninja-build bison nasm binutils clang libgles2-mesa-dev \
+        mesa-common-dev libpulse-dev libasound2-dev libxrender-dev \
+        libxcomposite-dev libxml2-dev curl git python3.11-venv libxi-dev \
```
Comment (kaidokert) [inline cobalt/site/docs/development/setup-linux.md:24]: The page above mentions "These instructions were tested on a fresh ubuntu:20.04 Docker image" - can we validate this runs ? Should be as easy as checking with `docker run ubuntu:20.04`

Comment (kaidokert) [reply cobalt/site/docs/development/setup-linux.md:24]: Specifically, python3.x should match up.  We need higher than 3.7 but whatever comes from that particular image should match.

**Review by alexanderahn — APPROVED** [review]
No TW comments from me, LGTM

File: cobalt/site/docs/development/setup-linux.md (hunk 1 above)
Comment (oxve) [reply cobalt/site/docs/development/setup-linux.md]: I kept the original version here instead. We do support 3.8 and on ubuntu 20.04 only 3.9 is available. With the disclaimer above the instructions I believe we are good.

Comment (oxve) [reply cobalt/site/docs/development/setup-linux.md]: I should add that I tested the entire flow on a clean ubuntu 20.04 and got it all to work.

**Review by dahlstrom-g — COMMENTED** [review]
Whoa, did GitHub never show these comments I wrote yesterday
except to me? Oops. 😕

File: cobalt/site/docs/development/setup-linux.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ Required libraries can differ depending on your Linux distribution and version.
...
+        ccache build-essential libglib2.0-dev
```
[the remark this answers is not in the document]
Comment (oxve) [reply cobalt/site/docs/development/setup-linux.md]: Trimmed the package list based on your suggestion. Also updated the instructions for ccache to point out that it's required by default and must be disabled if not installed.

File: cobalt/site/docs/development/setup-docker.md
[the remark this answers is not in the document]
Comment (oxve) [reply cobalt/site/docs/development/setup-docker.md:1]: I've verified all the steps using a clean ubuntu docker image

**Review by kaidokert — APPROVED** [review]

**Pull request merged**